python/res/enkf/jobs/scaling/scaling_job.py

```diff
@@ -219,7 +220,8 @@ def has_data(observations, keys, ensamble_size, storage):
     Checks that all keys have data and returns a list of error messages
     """
     error_msg = "Key: {} has no data"
-    active_mask = BoolVector(True, ensamble_size)
+    active_realizations = SummaryCollector.createActiveList(ensamble_size, storage)
```

Comment:
It seems a bit arbitrary to get this functionality from the `SummaryCollector`. How about using `storage.realizationList(RealizationStateEnum.STATE_HAS_DATA)` instead?

Reply:
That makes more sense :+1: